config_parser/config_test.go

```diff
@@ -62,6 +64,25 @@ func Test_ParseConfig(t *testing.T) {
 		{
 			name:     "test_process_returns_expected_config_on_additional_data_in_yaml_file",
 			filePath: "test4.yaml",
+			expectedConfig: &Config{
+				Assemblers: AssemblerConfig{
+					Megahit: MegahitConfig{KMers: "27"},
+					Abyss:   AbyssConfig{KMers: "27"},
+					Flye:    FlyeConfig{},
+				},
+				GenoMagic: GenoMagicConfig{
+					Assemblers:    []string{"abyss", "megahit", "flye"},
```

Comment:
How likely is it that someone will not abide by this format? Specifying "abyss" instead of "Abyss", or "abYss"?

Reply:
1. This commit (3324980) adds appropriate documentation in `README.md` file about assemblers.
2. This commit (https://github.com/genomagic/genomagic/commit/329bfd36510c639f0f363922c67bb842b1d5dbc9) already ensures that assemblers matching the case get executed: